### ISSUE TITLE
CondaHTTPError: HTTP 000 CONNECTION FAILED

### ISSUE THREAD

**Issue by ialbert** [biostars/biostar-handbook-issues#24]
Conda fails with:

```
 CondaHTTPError: HTTP 000 CONNECTION FAILED for url https://conda.anaconda.org/biostar/linux-64/repodata.json
    Elapsed: -
    
```
    An HTTP error occurred when trying to retrieve this URL.
    HTTP errors are often intermittent, and a simple retry will get you on your way.
    SSLError (MaxRetryError('HTTPSConnextionPool(host='conda.anaconda.org', port 443):Max retries exceeded with yrl: /bioconda/linux-64/repodata.json(Caused by SSLError (''Can't connect to HTTPS URL because the SSL module is not available. '',))',),)

**Comment by ialbert** [comment]
Fix it with:

    sudo apt install execstack
    execstack -c ~/miniconda3/lib/libcrypto.so.1.0.0

Related to: https://github.com/ContinuumIO/anaconda-issues/issues/6424

**Comment by ialbert** [comment]
I just got a note and confirmation that the latest conda does not work on windows.

See the conda installation on Windows (I have just updated it) make sure to reload your page.

https://www.biostarhandbook.com/setup/setup-conda.html

In a nutshell you will need to remove your existing conda package with

    rm -rf miniconda3

then install a previous version of it like so:

    wget https://repo.continuum.io/miniconda/Miniconda3-4.3.21-Linux-x86_64.sh
    bash Miniconda3-4.3.21-Linux-x86_64.sh

Now if conda is not visible in your terminal it means you have not installed it properly.
